pkg/kubeapiserver/admission/config.go

```diff
@@ -19,23 +19,17 @@ package admission
 import (
 	"net/http"
 	"os"
-	"time"
 
 	"k8s.io/klog/v2"
 
 	"go.opentelemetry.io/otel/trace"
 
-	utilwait "k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/apiserver/pkg/admission"
 	webhookinit "k8s.io/apiserver/pkg/admission/plugin/webhook/initializer"
-	genericapiserver "k8s.io/apiserver/pkg/server"
-	"k8s.io/apiserver/pkg/server/egressselector"
+	egressselector "k8s.io/apiserver/pkg/server/egressselector"
```

Comment:
```suggestion
	 "k8s.io/apiserver/pkg/server/egressselector"
```
nit. probably rebase issue.

Reply:
Yes it comes from rebasing^ removed